package/libs/libevent2/Makefile

```diff
@@ -8,13 +8,16 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libevent2
-PKG_VERSION:=2.0.22
-PKG_RELEASE:=1
+PKG_VERSION:=2.1.8
+PKG_RELEASE:=2
```

Comment:
I will cooperate with @azat to test cmake of libevent, after finish that task, I will update this PULL REQUEST.